src/components/text-area/TextArea.js

```diff
@@ -40,18 +44,33 @@ const TextArea = ({
         'show-error': hasError,
     });
 
+    const textAreaID = React.useRef(id || uniqueId('textArea')).current;
     const errorMessageID = React.useRef(uniqueId('errorMessage')).current;
+    const descriptionID = React.useRef(uniqueId('description')).current;
+
     const ariaAttrs = {
         'aria-invalid': hasError,
         'aria-required': isRequired,
         'aria-errormessage': errorMessageID,
+        'aria-describedby': description ? descriptionID : undefined,
     };
 
     return (
         <div className={classes}>
-            <Label hideLabel={hideLabel} showOptionalText={!hideOptionalLabel && !isRequired} text={label}>
+            <Label
+                hideLabel={hideLabel}
+                showOptionalText={!hideOptionalLabel && !isRequired}
+                text={label}
+                htmlFor={textAreaID}
```

Comment:
`htmlFor` can be removed if we keep nesting the form.

Reply:
👍